src/setup/cluster.rst

```diff
@@ -31,10 +31,11 @@ CouchDB uses the following ports:
 |             |          | default ``localhost`` | port for all HTTP    |
 |             |          |                       | API requests         |
 +-------------+----------+-----------------------+----------------------+
-| 4369        | tcp      | All interfaces        | Erlang port mapper   |
-|             |          | by default            | daemon (epmd)        |
+| 4369        | tcp      | localhost for single  | Erlang port mapper   |
+|             |          | node installs. private| daemon (epmd)        |
```

Comment:
Tiny nit: Capitalize `P` for consistency with  other rows. Use ``` ``localhost`` ``` like in the row above.